### PR TITLE
feat: add inputs argument

### PR DESCRIPTION
**Pull request by ghost**
This may help @nlordell.

Originally I thought `params` would be a good arg, but it wouldn't accept it due to `prefix`.

### CODE REVIEW

**Comment by ghost** [comment]
If you're happy with approach, I can add updated documentation.

File: cli/src/main.rs [hunk 1]
```diff
@@ -51,6 +51,12 @@ struct Args {
     /// Only output the transaction calldata without any extra information.
     #[arg(short, long)]
     quiet: bool,
+
+    /// Quiet mode.
+    ///
+    /// Only output the transaction calldata without any extra information.
+    #[arg(short, long)]
+    inputs: bool,
```
Comment (nlordell) [inline cli/src/main.rs]: I liked your suggestion of calling this `params` to print out the `createProxyWithNonce` parameters.

The issue with it not working with the existing `prefix` flag is that `#[arg(short)]` defaults to using the first letter of there field name. You can fix that by specifying something else, for example:

```suggestion
    #[arg(short = 'P', long)]
    params: bool,
```

This way, `-P` is used to print parameters, and `-p` is used for setting a prefix.

**Comment by nlordell** [comment]
Changes and approach looks good! Thanks for taking care of this :) Just a comment around the parameter name.

**Comment by ghost** [comment]
Hey @nlordell 

Changed as advised.

Updated the docs, included a note about Etherscan.

There are two more commits:

1. To me it makes sense for creating the safe to be above Unsupported Chains, as it's the more common immediate usage
2. I suggest adding a note about safe 1.3.0. Currently it seems safe-service doesn't automatically pick up on the creation event from 1.4.1 safes - unless I'm understanding incorrectly, so it seems like a good idea to guide people to 1.3.0 vanity generation?

I realise now these should probably be different pull requests. Happy either way as this is your repo :)

File: README.md [hunk 2]
```diff
@@ -96,6 +96,17 @@ Send a `0eth` transaction to the factory address, placing the generated calldata
...
+
+Use the `--params` flag to output contract-ready inputs.
+
+1. Visit the [factory address](https://etherscan.io/address/0x4e1DCf7AD4e460CfD30791CCC4F9c8a4f820ec67).
+2. Click Contract -> Write Contract -> Connect to Web3.
+3. Connect the account you wish to pay for the Safe creation.
+
+Fill the fields in function `3. createProxyWithNonce` using the generated outputs.
+
+
```
Comment (nlordell) [inline README.md:111]: nit: extra newline
```suggestion
```

**Comment by nlordell** [comment]
> To me it makes sense for creating the safe to be above Unsupported Chains, as it's the more common immediate usage

100% agree.

> Currently it seems safe-service doesn't automatically pick up on the creation event from 1.4.1 safes

Oh no! Then yes, you should probably roll back to using v1.3.0 Safes for now. I also just double checked ant the wallet interface creates a v1.3.0 Safe as well, so it looks like v1.4.1 is not super well supported yet. Thanks for bringing this up!

**Comment by nlordell** [comment]
@kennedybaird - I rolled back the CLI to use Safe v1.3.0 because of compatibility concerns with the Safe infrastructure and Wallet interface. Thanks a bunch for bringing this up!

**Comment by kennedyatfeature** [comment]
That's great, yeah, I ended up down a bit of a rabbit hole before figuring that out.

Good work!

**Comment by ghost** [comment]
That's great, yeah, I ended up down a bit of a rabbit hole before figuring
that out.

Good work!

**Pull request merged**